### ISSUE TITLE
System won’t update to 3.5.5 after numerous attempts 

### ISSUE THREAD

**Issue by Sniperinthesky1** [ValveSoftware/SteamOS#1213]
#### Your system information

* Steam client version: 1700160213
* SteamOS version: 3.4.11
* Opted into Steam client beta?: No 
* Opted into SteamOS beta?: No 
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible: Have tried multiple times at random times throughout the days since it was released. The things I’ve tried I listed below. But yet every time it tries to update. It gets to 100% and then a message pops up and says “Update Error. An error was encountered during the update process. Please try again.”

Things I’ve tried:

updated everything in Discovery on Desktop Mode
tried updating on phone hotspot
turned off fully and restarted
tried during different times of the day hoping the servers weren’t messed up
I do have Windows 10 as a dual boot option on my system but still have plenty of space on my internal ssd, like 82GB free out of 140.2GB. I don’t have cryoutilities installed.

Describe what you _expected_ should happen and what _did_ happen.

#### Steps for reproducing this issue:

1. Check for updates
2. Click Apply Update
3. System then tells me Update Error

**Comment by RyuzakiKK** [comment]
Please try to apply an update again, and after it fails, create a system report by going into Settings -> System and near the bottom you have the option to submit a system report.

**Comment by Sniperinthesky1** [comment]
> Please try to apply an update again, and after it fails, submit a system report. The option to create and submit a system report is in Settings -> System and you'll find it near the bottom of it.

Just went in and submitted a system report

**Comment by RyuzakiKK** [comment]
@Sniperinthesky1 Thanks. I'm sorry but I forgot to mention that you should also post a link to your Steam profile (or its username) in order to allow the Steam support to know which report is yours.

**Comment by RyuzakiKK** [comment]
Until we know more, @kisak-valve could you please re-title this with something like "3.5.5 update fails after reaching 100%". In this way we'll avoid turning this issue into a catch-all for every 3.5.5 update problem.

Maybe even adding a label "Need More Information" would be useful, because we can't proceed any further here until we have the user's Steam profile link.

**Comment by RyuzakiKK** [comment]
@Sniperinthesky1 From your logs it seems like the system update system is attempting to update the other A/B partition, but it seems like it is failing because it unexpectedly contains a Windows installation instead.

Could you please post the output of:
- `rauc status`
- `lsblk`
- `sudo parted -l`

Unfortunately it seems like your Windows installation has wrongfully altered the expected SteamOS partitions.

**Comment by Sniperinthesky1** [comment]
> @Sniperinthesky1 From your logs it seems like the system update system is attempting to update the other A/B partition, but it is failing because it unexpectedly contains a Windows installation instead.
> 
> Could you please post the output of:
> 
> * `rauc status`
> * `lsblk`
> * `sudo parted -l`
> 
> Unfortunately it seems like your Windows installation has wrongfully altered the expected SteamOS partitions.

Where might I find these outputs?

**Comment by RyuzakiKK** [comment]
Those are Linux commands. You can switch to desktop mode, open the "konsole" application and type those commands one by one. If you never set a root password to use `sudo` before, you can skip `sudo parted -l`.

Those will give us a more clear view of what the partitions look like in your system but to be honest it's unlikely that this state is fixable without formatting.

Your best option is to follow the [SteamOS recovery instructions](https://help.steampowered.com/en/faqs/view/1b71-edf2-eb6d-2bb3) and start from a sane clean state again by using "Re-image Steam Deck".

**Comment by Sniperinthesky1** [comment]
@RyuzakiKK yeah I’ve been just considering doing a factory reset of doing the steam reimage and making SteamOS my sole OS again. Having Windows as a side partition was nice. But realistically I was only using it for one game and I’d prefer having a cleaned up Steam Deck again.
